daemon/mgr/container.go

```diff
@@ -540,6 +540,12 @@ func (mgr *ContainerManager) stoppedAndRelease(id string, m *ctrd.Message) error
 	c.Lock()
 	defer c.Unlock()
 
+	// lookup again, avoid container already removed
+	c, err = mgr.container(id)
+	if err != nil {
+		return err
```

Comment:
need to handle the error. if Notfound, return nil.

Reply:
We should move the diff to 565 line after closing IO, because the IO must be closed to release resource.

Reply:
ok, thanks for your guidance. :-)